invenio_records_resources/services/files/service.py

```diff
@@ -79,15 +78,13 @@ def list_files(self, id_, identity):
             links_item_tpl=self.file_links_item_tpl(id_),
         )
 
-    def init_files(self, id_, identity, data):
+    @unit_of_work()
```

Comment:
I like that we call it as what it does, but I am wondering if it should be maybe more specific? E.g. `commit_after` or `persist_after` (sorry, no better naming at the moment...)
At the same time, such name reminds me of [invenio-deposit](https://github.com/inveniosoftware/invenio-deposit/blob/master/invenio_deposit/api.py#L187), and we never liked that...